src/app/App.js

```diff
@@ -14,4 +16,20 @@ function App() {
     );
 }
 
-export default App;
+const mapStateToProps = (state) => {
+  return {
+    currMovie: state.currMovie,
+    moviesList: state.moviesList,
+  };
+};
+
+const mapDispatchToProps = (dispatch) => ({
+  reset(){
+    dispatch(ActionCreator.reset());
+  },
+
+});
+
+
+
+export default connect(mapStateToProps, mapDispatchToProps)(App);
```

Comment:
Instead of mapDispatchToProps you can use export default connect(mapStateToProps, {ActionCreator.reset})(App);